app/websocket/app/main/model_manager.py

```diff
@@ -0,0 +1,106 @@
+import syft as sy
```

Comment:
As discussed on slack we should use a proper db (SQLAlchemy, postgres, ...) not a file.

Reply:
Yup that’s the plan will replace files with db now. 
Just wanted to give it a structure.

Reply:
added db.. integrated with @IonesioJunior sqlalchemy implementation.